autorag/utils/util.py

```diff
@@ -310,3 +312,13 @@ def sort_by_scores(row, reverse=True):
     results = sorted(zip(row['contents'], row['ids'], row['scores']), key=lambda x: x[2], reverse=reverse)
     reranked_contents, reranked_ids, reranked_scores = zip(*results)
     return list(reranked_contents), list(reranked_ids), list(reranked_scores)
+
+
+def get_cosine_similarity_scores(queries: List[str], contents_list: List[List[str]]) -> List[List[float]]:
```

Comment:
We already have `calculate_cosine_similarity` function in `autorag/evalaute/metrics/util.py`
It looks like this function do the same, but got different embedding. 
Why don't you use that? You can change the function's location.